README.md

```diff
@@ -1,2 +1,76 @@
-# ts_m2fpga
-M2 Controller FPGA code
+# M2 Cell Controller FPGA Code
+
+## Overview
+
+This project will improve the FPGA code in the [M2 Cell Controller LaVIEW project](https://github.com/lsst-ts/ts_mtm2_cell).
+In that version, the VI named **4-port serial Master Slave.vi**, under `Chassis (cRIO-9038) >> FPGA Target (RIO0, cRIO-9038)` in the Project Explorer, interacts with the hardware inputs and outputs.
+
+## Directory Structure
+
+```bash
+ts_m2fpga
+├── src
+│   ├── m2fpga.lvproj
+│   ├── mainFPGA.vi
+│   ├── moduleRelays.vi
+│   ├── moduleDaq.vi
+│   ├── moduleIlc.vi
```

Comment:
It is weird to me that you have the vi named begin from `module`. It is a reserved word with the specific meaning at the most of time. Maybe `commRelays.vi`, `commDaq.vi`, and `commIlc.vi`?

Reply:
Changed: 
- `moduleRelays.vi` --> `switchRelays.vi`
- `moduleDaq.vi` --> `readDaq.vi`
- `moduleIlc.vi` --> `ilcCommunication.vi`

Reply:
Done.